packages/fluentui/react-northstar/src/themes/teams/components/Pill/pillActionStyles.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { PillActionStylesProps } from '../../../../components/Pill/PillAction';
+import { PillVariables } from './pillVariables';
+import { svgIconClassName } from '@fluentui/react-icons-northstar';
+
+export const pillActionStyles: ComponentSlotStylesPrepared<PillActionStylesProps, PillVariables> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    border: 'none',
+    background: 'transparent',
+    marginLeft: v.actionPadding,
+    marginRight: v.actionPadding,
+    width: v.actionWidth,
+    cursor: 'pointer',
+
+    ...(p.size !== 'medium' && {
```

Comment:
Can we avoid negative condition?

```suggestion
    ...((p.size === 'small' || p.size === 'smaller') && {
```